shortfin/src/shortfin/components/tokenizers/CMakeLists.txt

```
@@ -0,0 +1,39 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+if(SHORTFIN_ENABLE_TOKENIZERS)
+    shortfin_cc_component(
+      NAME
+        shortfin_tokenizers
+      HDRS
+        tokenizers.h
+      SRCS
+        tokenizers.cc
+      DEFINES
+        SHORTFIN_HAVE_TOKENIZERS
+      COMPONENTS
+        shortfin_support
+      DEPS
+        tokenizers_cpp
+    )
+    set_property(GLOBAL APPEND
+      PROPERTY SHORTFIN_LIB_OPTIONAL_COMPONENTS
+      shortfin_tokenizers)
+    target_compile_definitions(shortfin_public_defs INTERFACE SHORTFIN_HAVE_TOKENIZERS)
+
+    # Download test data.
+    shortfin_download_test_data(
+      URL "https://huggingface.co/google-bert/bert-base-cased/resolve/main/tokenizer.json"
+      OUTPUT_FILE "${CMAKE_CURRENT_BINARY_DIR}/tokenizer.json"
+    )
```

I assume this file doesn't need authentication to download?

Looks like the file is 426KB. Might be small enough to check in instead of fetch from the network...

also nit: could pin to a specific commit hash to stabilize the supply chain a bit:
```suggestion
    # Download test data.
    shortfin_download_test_data(
      URL "https://huggingface.co/google-bert/bert-base-cased/resolve/cd5ef92a9fb2f889e972770a36d4ed042daf221e/tokenizer.json"
      OUTPUT_FILE "${CMAKE_CURRENT_BINARY_DIR}/tokenizer.json"
    )
```

---

Yeah, it's a purely public one that others seem to use a lot. But I couldn't find anyone else who wasn't downloading it... decided to go with the crowd.

---

Pinned it as suggested.